docs/features/software-templates/authorizing-scaffolder-template-details.md

```diff
@@ -176,7 +176,9 @@ class ExamplePermissionPolicy implements PermissionPolicy {
 
 ### Authorizing scaffolder tasks
 
-The scaffolder plugin also exposes permissions that can restrict access to tasks, task logs, task creation, and task cancellation. This can be useful if you want to control who has access to these areas of the scaffolder.
+The scaffolder plugin also exposes permissions that can restrict access to tasks, task logs, task creation, and task cancellation. This can be useful if you want to control who has access to these areas of the scaffolder. You can also restrict access to tasks to allow template owners to view all runs of their templates.
```

Comment:
Can we make a reference to [here](https://backstage.io/docs/features/software-templates/#template-task-list) just so its clear what we're referring to? Would be good I think! WDYT?